vfh:
  LA: 10
  wg: 10
  wd: 1
  wo: 7
  crit_cost: 75
  ndpsi: 72